src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployBundleService.kt

```diff
@@ -103,6 +104,28 @@ class DeployBundleService(
         return createAuroraDeploymentSpec(auroraConfig, applicationId, dockerRegistry, overrideFiles, vaults)
     }
 
+    fun validateAuroraConfig(affiliation: String, function: (AuroraConfig) -> AuroraConfig = { it -> it }): AuroraConfig {
+        val deployBundle = createDeployBundle(affiliation)
+        val auroraConfig = deployBundle.auroraConfig
+
+        val newAuroraConfig = function(auroraConfig)
+        deployBundle.auroraConfig = newAuroraConfig
+        validate(deployBundle)
```

Comment:
Since there are now more than one method doing validation, the validate(deployBundle) should probably be renamed to use the same naming scheme of the other method (validateAuroraConfig).